packages/terra-functional-testing/lib/config/wdio.conf.js

```diff
@@ -80,6 +82,8 @@ exports.config = {
   // If you only want to run your tests until a specific amount of tests have failed use
   // bail (default is 0 - don't bail, run all tests).
   bail: 0,
```

Comment:
Did we need add the hostname & port to the config here? [or am I missing the default values?](https://github.com/cerner/terra-toolkit/pull/412/files?file-filters%5B%5D=.js&file-filters%5B%5D=.json&file-filters%5B%5D=dotfile#diff-baeaa8efd5bd180c14c546899ee9daf3R18-R19)
```
hostname: `http://localhost`,
seleniumPort: 4444,
```

Reply:
Added the hostname and port:

https://github.com/cerner/terra-toolkit/pull/412/commits/5081644beac4fd9e527541810ddcd4c9c934531e

Note: hostname was added as `localhost` without the http protocol. This doesn't appear to cause any issues. I took the defaults as listed on the WebDriverIO doc site:

https://webdriver.io/docs/options.html